lib/about.js

```diff
@@ -17,6 +17,7 @@ define(function () {
         '<span class="legend-5ghz"><span class="symbol"></span> 5 GHz</span>' +
         '<span class="legend-others"><span class="symbol"></span> ' + _.t('others') + '</span>' +
         '</p>' +
+        ( devicePicturesAttribution ? ('<h3>Device Pictures Attribution</h3><p>' + devicePicturesAttribution + '</p>') : '') +
```

Comment:
The text should be translatable by using `_.t('[...]')`

Reply:
I don't really know how I can achieve both: translatable and configurable.. But the former is probably more useful..?

Reply:
it would become quite ugly..
```
if devicePictures is set, show the attribution
    if attribution text is set, show the attribution text, else show the translatable text
else do not show the attribution
```

thats 4 different cases for this.. Well - i will remove the configurability, as most communities will hopefully use the upstreamed pictures if using this feature.

Reply:
It could eventually be made shorter to two config options: link to images and license name? So something like `[Pictures](linked) licensed under [license]` or something (maybe better wording)?